### ISSUE TITLE
Escaped jinja syntax in SQL query cannot be formatted

### ISSUE THREAD

**Issue by alvarezej** [CyberShadow/squelch#2]
Jinja escaped string cannot be formatted by Squelch. I am trying to generate a (BigQuery + Dbt)  query using jinja.

Example query:

```SQL
WITH
subquery AS (
  SELECT
    field1,
    field2,
    field3,
    field4 AS {{ name }}
  FROM
  {{ "{{ ref('table_reference') }}" }})

SELECT * FROM subquery
```

example companion data for that jinja template

```YAML
name: customer_id
```

Rendering the above example query in jinja  produces the desired output by correctly escaping the line `{{ "{{ ref('table_reference') }}" }})` . Producing the following desired output:

```SQL
WITH
subquery AS (
  SELECT
    field1,
    field2,
    field3,
    field4 AS {{ name }}
  FROM
  {{ ref('table_reference') }})

SELECT FROM subquery
```

Formatting this query e.g. using https://squelch-sql-formatter.herokuapp.com/ returns the error

```
Fatal error: Unterminated string
```

----

Trying with `%raw%` tag Squelch is able to format the query

```SQL
WITH
  subquery AS (
    SELECT
      field1,
      field2,
      field3,
      field4 AS {{ name }}
    FROM
      {% raw %}
      {{ ref('table_reference') }}
  )
  {% endraw %}
SELECT
  *
FROM
  subquery
```

From the jinja [docs](https://jinja.palletsprojects.com/en/3.1.x/templates/#escaping), both methods should work. Is this something that could be supported? Otherwise, large queries can quickly get cluttered by these `%raw%` tags. Specially since Dbt relies on this macro for referencing tables.

Thanks for creating this awesome and blazing fast formatter! @CyberShadow @MartinNowak

**Comment by CyberShadow** [comment]
Fixed in v0.2.4, thanks for the report.

The web formatter is at a dead-end due to Heroku removing their plan, will need to move it somewhere else.

**Comment by alvarezej** [comment]
Thank you for implementing this. I tested it using  [v0.2.4](https://github.com/CyberShadow/squelch/releases/tag/v0.2.4) and works like a charm. Closing this.
